plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -230,9 +233,9 @@ ownStakePubKeyHash skh = mempty { slOwnStakePubKeyHash = Just skh }
 -- | An unbalanced transaction. It needs to be balanced and signed before it
 --   can be submitted to the ledger. See note [Submitting transactions from
 --   Plutus contracts] in 'Plutus.Contract.Wallet'.
-data UnbalancedTx =
-    UnbalancedTx
-        { unBalancedTxTx                  :: Either C.CardanoBuildTx Tx.Tx
+data UnbalancedTx
+    = UnbalancedLedgerTx
```

Comment:
`UnbalancedLedgerTx` sounds like it's a transaction from `cardano-ledger`. `UnbalancedEmulatorTx` is probably a better name here.